core/tests/protractor_utils/creatorDashboard.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Oppia Authors. All Rights Reserved.
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
```

Comment:
Let's maybe call this CreatorDashboardPage.js, and the object CreatorDashboardPage. That makes it easy to distinguish files that have been fixed from files that haven't been -- everything that is a page object ends with "Page". It also brings this in line with our other JS services which are transitioning to UpperCamelCase.

Reply:
Done.Done.

Reply:
Done.